### ISSUE TITLE
Accessibility - Poor Color Contrast for A - Z List on Select Your Cities Screen

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Closure-App#13]
The Select Your Cities screen has poor color contrast for A - Z list. the light blue text has a color contrast ratio of 3.29:1 and needs to be 4.5:1 or higher to pass. 

You can make the text black (#000000) or use dark gray (#3d3d3d) to have enough color contrast. You should make sure all the other color contrast issues use the same color for your black or dark gray. Doing so will help people to read it since the text is so small.

See screenshot below.

![closure - poor color cntrast for select your cities a - z list](https://user-images.githubusercontent.com/10423487/34885875-5195f27e-f78f-11e7-9e57-4e940983e359.png)

**Comment by georgepliu** [comment]
The color is now black.  This issue has been resolved and is reflected in the current build #11

**Comment by jfc3-dol** [comment]
Thanks, for making the text black to improve the color contrast enough to pass.

I'm closing this issue.
